client/src/components/ObjectStore/ConfigurationMarkdown.vue

```diff
@@ -0,0 +1,18 @@
+<script setup lang="ts">
+import { computed } from "vue";
+import { markup } from "./configurationMarkdown";
+interface Props {
+    markdown: string;
+    admin: boolean; // was the configuration specified by an admin
+}
+const props = defineProps<Props>();
+const markdownHtml = computed(() => markup(props.markdown || "", props.admin));
```

Comment:
```suggestion
const markdownHtml = computed(() => markup(props.markdown ?? "", props.admin));
```

Very minor but just to be consistent with the [Styleguide](https://github.com/galaxyproject/galaxy/blob/dev/client/docs/styleguide.md#default-value-assignment) :wink:

Reply:
Very nice - thanks for the link! That is really good to know and I didn't know the difference!